Public/Src/Cache/MemoizationStore/Library/Sessions/LocalCache.cs

```diff
@@ -35,16 +37,29 @@ public class LocalCache : OneLevelCache
             ConfigurationModel configurationModel = null,
             IClock clock = null,
             bool checkLocalFiles = true,
-            bool emptyFileHashShortcutEnabled = false)
+            bool emptyFileHashShortcutEnabled = false,
+            bool createServer = false,

```

Comment:
Need a settings class IMO. Or if you'll add one that I've suggested already, maybe we can just pass it here.